app/code/Magento/Catalog/Model/Config.php

```diff
@@ -64,6 +64,13 @@ class Config extends \Magento\Eav\Model\Config
      */
     protected $_usedForSortBy;
 
+    /**
+     * Product Attributes For Price Rules
+     *
+     * @var array
+     */
+    protected $_usedForPriceRules;
```

Comment:
1. Please change property visibility from protected to private
2. Please remove underscore for variable name